src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+
```

Comment:
Delete this line you should add an empty line when there are sibling multiline blocks, here you have sibling blocks,  but each of them are in one line so you can remove this empty line everywhere